src/dex/woo-fi-v2/woo-fi-v2-math.ts

```diff
@@ -97,6 +113,14 @@ export class WooFiV2Math {
       let state1: TokenState = this.state.tokenStates[baseToken1];
       let state2: TokenState = this.state.tokenStates[baseToken2];
 
+      if (!state1.woFeasible || !state2.woFeasible) {
+        const token = !state1.woFeasible ? baseToken1 : baseToken2;
+        this.logger.error(
+          `Oracle state for base token ${token} is not feasible`,
+        );
+        return null;
```

Comment:
Both tokens can have `woFeasible` false, would be nice to log all of them

Reply:
done